python/py-plotly/Portfile

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-plotly
+version             5.3.1
+revision            0
+
+categories          python graphics
+platforms           darwin
+supported_archs     noarch
+license             MIT
+maintainers         {nist.gov:joe.fowler @joefowler} openmaintainer
+
+description         Interactive graphing library for Python.
+long_description \
+    Plotly's Python graphing library makes interactive, publication-quality graphs. Examples of how to \
+    make line plots, scatter plots, area charts, bar charts, error bars, box plots, histograms, \
+    heatmaps, subplots, multiple-axes, polar charts, and bubble charts. 
```

Comment:
can you please fix the lint warning? 
```
port lint --nitpick py-plotly:
Warning: Line 20 has trailing whitespace before newline
```

Reply:
Right. Didn't know about `--nitpick`, but now I do. Improved commits coming soon...